doc/faq.rst

```diff
@@ -0,0 +1,54 @@
+Frequently asked questions
+==========================
+
+Should I apply ICA first or autoreject first?
+---------------------------------------------
+
+ICA solutions can be affected by high amplitude artifacts, therefore
+we recommend first using autoreject to detect the bad segments, then applying
+ICA, and finally interpolating the bad data.
+
+To ignore bad segments using autoreject (local), we could do::
+
+	>>> ar = Autoreject()
+	>>> _, reject_log = ar.fit(epochs).transform(epochs)
```

Comment:
this is not correct as `return_log=False` by default.

it should be:

`_, reject_log = ar.fit_transform(self, epochs, return_log=True)``